homeassistant/components/home_connect/binary_sensor.py

```diff
@@ -26,11 +31,27 @@ def get_entities():
 class HomeConnectBinarySensor(HomeConnectEntity, BinarySensorEntity):
     """Binary sensor for Home Connect."""
 
-    def __init__(self, device, desc, device_class):
+    def __init__(self, device, desc, sensor_type, device_class=None):
         """Initialize the entity."""
         super().__init__(device, desc)
-        self._device_class = device_class
         self._state = None
+        self._device_class = device_class
+        self._type = sensor_type
+        if self._type == "door":
+            self._update_key = BSH_DOOR_STATE
+            self._false_value_list = [
+                "BSH.Common.EnumType.DoorState.Closed",
+                "BSH.Common.EnumType.DoorState.Locked",
+            ]
```

Comment:
Since it's never changed, a tuple is fine. 
```suggestion
            self._false_value_list = (
                "BSH.Common.EnumType.DoorState.Closed",
                "BSH.Common.EnumType.DoorState.Locked",
            )
```